library/src/main/java/com/ibm/research/drl/dpt/anonymization/constraints/EntropyLDiversity.java

```diff
@@ -107,4 +107,9 @@ public void initialize(IPVDataset dataset, List<ColumnInformation> columnInforma
         sanityChecks(dataset);
     }
 
+    @Override
```

Comment:
same here, should upgrade copyright to 2022?

Reply:
No real changes to this file